src/lib/mirage/utils/create-tauri-route.ts

```diff
@@ -0,0 +1,16 @@
+import { Response } from "miragejs";
+
+type BodyParamType = ConstructorParameters<typeof Response>[2];
+
+export function createTauriResponse(body: BodyParamType) {
+  return new Response(
+    200,
+    {
+      "Access-Control-Allow-Origin": "*",
+      "Access-Control-Expose-Headers": "Tauri-Response",
+      "Content-Type": "application/json",
+      "Tauri-Response": "ok",
+    },
```

Comment:
Does tauri use classic HTTP header/body to communicate between front and back ?

Reply:
Yep! My process was to look at what happens when I call the `invoke` function.

Turns out it's a plain HTTP POST request. So I mocked it with miragejs.

The request was indeed being intercepted, but it was giving an error, which was odd, because miragejs was definitely giving a 200 response.

Turns out, `invoke` has some under-the-hood validation, and they need some special headers (most importantly, Tauri-Response set to ok).